03-Binary-Search-Trees/02-Binary-Search-Trees.md

```diff
@@ -961,7 +961,7 @@ Consider creating a helper function to find the minimum node in a tree.
 
 Still feeling stuck? Check this video walkthrough of the solution.
 
-<!-- ADD VIDEO WALKTHROUGH -->
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?id=5e4deb73-094d-4c97-b2db-af0e0148f7f9&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&captions=true&interactivity=all" height="360" width="640" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
```

Comment:
- In the diagram walkthrough, specify 'the left subtree _of node 15_ is empty' 
- Consider calling out at the end of the diagram walkthrough, that finding the minimum node in the right subtree is a subproblem and thus is a good opportunity to write a helper function

- For a problem like this, perhaps we may consider including a whiteboard walkthrough at the end. Where we go through different small test cases and the implementation line by line so they can see how the code handles actual examples. With recursion I wonder if it may also help to draw the call stack.